ouster_client/src/client.cpp

```diff
@@ -300,12 +300,11 @@ std::shared_ptr<client> init_client(const std::string& hostname,
 
         // if specified (not UNSPEC), set the lidar and timestamp modes
         if (mode) {
-            sensor_http->set_config_param("lidar_mode", std::to_string(mode));
+            sensor_http->set_config_param("lidar_mode", to_string(mode));
```

Comment:
```suggestion
            sensor_http->set_config_param("lidar_mode", sensor::to_string(lidar_mode));
```

Reply:
:+1: we'll get it on the next one
(ditto for the other)